specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/expressRouteCircuit.json

```diff
@@ -1863,11 +1863,12 @@
         },
         "tier": {
           "type": "string",
-          "description": "The tier of the SKU. Possible values are 'Standard', 'Premium' or 'Basic'.",
+          "description": "The tier of the SKU. Possible values are 'Standard', 'Premium' or 'Local'.",
```

Comment:
Is Basic being replaced? Please fix description.

Reply:
Yes, Basic is being replaced.

Reply:
Please fix the enum in that case. Also, this will be a breaking change

Reply:
Note that we will be deprecating Basic. Hence, its removed from the description. However, for backward compatibility, we want to keep Basic in the enum as well.

Reply:
Why is this a breaking change?